llama_stack/providers/utils/inference/litellm_openai_mixin.py

```diff
@@ -112,9 +112,9 @@ async def chat_completion(
 
         params = await self._get_params(request)
         logger.debug(f"params to litellm (openai compat): {params}")
-        # unfortunately, we need to use synchronous litellm.completion here because litellm
-        # caches various httpx.client objects in a non-eventloop aware manner
```

Comment:
Does this address the comment here about why it was using the sync API of litellm? What are we doing to ensure we're not running into litellm issues around caching of httpx.client objects in a way that's not compatible with event loops?

Reply:
yeah, thats a good shout
I tried to see if the async implementation already existed when this comment was made but i didin't end up following through 
i think @ashwinb might've been the one who added that comment, so a little more context here would be great if possible!